docs/docs/aztec/concepts/accounts/index.md

```diff
@@ -72,87 +63,99 @@ def entryPoint(payload):
         enqueueCall(to, data, value, gasLimit);
 ```
 
-Read more about how to write an account contract [here](../../../developers/tutorials/codealong/contract_tutorials/write_accounts_contract.md).
+A request for executing an action requires:
+- The `origin` contract to execute as the first step.
+- The initial function to call (usually `entrypoint`).
+- The arguments (which encode the private and public calls to run as well as any signatures).
 
-### Account contracts and wallets
-
-Account contracts are tightly coupled to the wallet software that users use to interact with the protocol. Dapps submit to the wallet software one or more function calls to be executed (e.g. "call swap in X contract"), and the wallet encodes and authenticates the request as a valid payload for the user's account contract. The account contract then validates the request encoded and authenticated by the wallet, and executes the function calls requested by the dapp.
+Read more about how to write an account contract [here](../../../developers/tutorials/codealong/contract_tutorials/write_accounts_contract.md).
 
-### Execution requests
+### Non-standard entrypoints
 
-Note that nothing related to signature verification or payload execution is enshrined in the protocol, since account contracts are free to define this entrypoint however they see fit. Therefore, a request for executing an action has a simpler structure than in Ethereum, and just requires:
+Since the `entrypoint` interface is not enshrined, there is nothing that differentiates an account contract from an application one. This allows implementing functions that do not need to be called by any particular user and are just intended to advance the state of a contract.
 
-- The `origin` contract to execute as the first step
-- The initial function to call (usually `entrypoint`)
-- The arguments (which encode the private and public calls to run as well as any signatures)
+For example, a lottery contract, where at some point a prize needs to be paid out to its winners. This `pay` action does not require authentication and does not need to be executed by any user in particular, so anyone could submit a transaction that defines the lottery contract itself as `origin` and `pay` as `entrypoint` function.
```

Comment:
You should add a note that the contract still needs to define how fees are paid, since they won't be paid by an account contract.